### DISCUSSION TITLE
How do I adjust the image adaptive size display in the report?

### DISCUSSION THREAD

**Discussion by 1997shp** [allure-framework/allure2#1569] [Questions & Support] [answered]
I'm using pytest + allure for Android automation testing
The mobile phone screenshot displayed in the report is too large
Is there any method or parameter to adjust the picture size
I hope to get your help

**Comment by baev** [comment, marked as answer]
as workaround you can use html attachments instead of png. Create a template html that will render image from base64 format (`<img src="data:image/png;base64,....">`) with desired styles
